opengm/plugins/base.py

```diff
@@ -57,3 +58,17 @@ async def help_button_callback(cl: Client, query: CallbackQuery) -> None:
         next_page = int(next_match.group(1))
         await query.message.reply(HELPTEXT, reply_markup=InlineKeyboardMarkup(paginate_modules(next_page + 1, HELPABLE, "help")))
     await query.answer()
+
+
+@Opengm.on_message(filters.command("reload") & filters.group)
+async def reload_admins(bot: Client, message: Message):
```

Comment:
Just coming here as a tourist as I am not familiar with this part of the code base yet: Is it really necessary to use this handler? Wouldn't it be more simple (from the point of the view of the user) to declare this functionality as part of the business logic when promoting / demoting people to / from the admin role?